packages/qwik-city/src/middleware/request-handler/request-event.ts

```diff
@@ -208,10 +207,6 @@ export function createRequestEvent(
         }
         headers.set('Location', fixedURL);
       }
-      headers.delete('Cache-Control');
-      if (statusCode > 301) {
```

Comment:
Can we add `headers.set('Cache-Control', 'no-store');` fallback if the header is not present and statusCode > 301 ?

Reply:
Thanks 👍